.markdownlint.yml

```diff
@@ -0,0 +1,33 @@
+---
+# MD007 - Unordered list indentation
+# Sets nested list indents to 4 spaces instead of the default of 2.
+# Required for proper ReadTheDocs rendering.
+MD007:
+    indent: 4
+
+# MD010 - No hard tabs
+# No hard tabs allowed in docs. Use 4 spaces instead.
+# This is disabled inline when used. E.g. docs/development/getting-started.md
+
+# MD013 - Line length
+# Defaults to 80 characters.
+MD013: false
+
+# MD014 - Dollar signs used before commands without showing output
+# Ignores terminal examples using dollar signs without any output examples after.
+# These occur in multiple places throughout our docs.
+MD014: false
+
+# MD033 - Inline HTML
+# Triggered when raw HTML is used in a markdown document
+# Used/disabled in a few files. E.g. docs/user-guides/graphql.md
+
+# MD041 - First line in a file should be a top-level heading
+# This requires a top-level heading as the first line in every doc.
+# This is only ignored inline in docs/index.md, but enabled globally elsewhere.
```

Comment:
I presume these are enabled by default, but it threw me for a bit of a loop on first reading because there's comments but no setting. Perhaps this might be clearer?

```suggestion
# MD033 - Inline HTML
# Triggered when raw HTML is used in a markdown document
# Used/disabled in a few files. E.g. docs/user-guides/graphql.md
MD033: true

# MD041 - First line in a file should be a top-level heading
# This requires a top-level heading as the first line in every doc.
# This is only ignored inline in docs/index.md, but enabled globally elsewhere.
MD041: true
```